alf/utils/common.py

```diff
@@ -570,26 +456,48 @@ def set_global_env(env):
 
 
 @gin.configurable
-def get_observation_spec(field=None):
+def get_raw_observation_spec(field=None):
```

Comment:
Do we still need this function? Typically gin files won't call it.

Reply:
Perhaps we don't need it anymore. But it doesn't hurt.